scrapy_autounit/commands.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+
+import os
+import sys
+from scrapy_autounit.utils import get_project_dir
+
+
+def update():
+    project_dir = get_project_dir()
+    if not project_dir:
+        print('No active Scrapy project')
+        sys.exit(1)
+
+    tests_dir = os.path.join(project_dir, 'autounit/tests')
+    for root, _, files in os.walk(tests_dir):
+        tests = [f for f in files if f.endswith('.py') and f != '__init__.py']
+        for test in tests:
+            test_path = os.path.join(root, test)
+            print('Updating test {}...'.format(test_path), end='')
+            with open(test_path, 'r+') as f:
```

Comment:
Could this just call `write_test`?  I expect all the required info's in the bin file.

Reply:
Oh, right! I thought about it but wasn't sure where to get the `encoding` parameter from.
Totally forgot we were recording that with the response. Changing this, thanks!

Reply:
Wait, to get the `encoding` I need to unpickle, and for that I need the encoding too 😅
I'll take a further look tomorrow.

Reply:
Ah, right :sweat_smile: good catch haha.

Yeah, that's tricky...

Still though I think scanning and modifying the .py file won't be realistic if there are more significant updates in the future...  Let me know if you don't have any ideas here and maybe we can brainstorm together.